src/extensions/renderer-api/catalog.ts

```diff
@@ -51,13 +51,15 @@ export class CatalogEntityRegistry {
   }
 
   /**
-   * Add a onBeforeRun hook to a catalog entity. If `onBeforeRun` was previously added then it will not be added again
-   * @param catalogEntityUid The uid of the catalog entity
-   * @param onBeforeRun The function that should return a boolean if the onRun of catalog entity should be triggered.
+   * Add a onBeforeRun hook to a catalog entities. If `onBeforeRun` was previously
+   * added then it will not be added again.
+   * @param onBeforeRun The function will receive the catalog run event as a parameter.
```

Comment:
How about this wording?
```suggestion
   * @param onBeforeRun The function to be called with a `CatalogRunEvent`
```